src/status_im2/contexts/wallet/send/events.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im2.contexts.wallet.send.events
+  (:require
+    [utils.number]
+    [utils.re-frame :as rf]))
```

Comment:
Nice you created a new namespace :+1: 
btw, I think `utils.number` is unused

Reply:
Thanks!